llvm/unittests/Target/DirectX/ResourceBindingAnalysisTests.cpp

```diff
@@ -167,7 +167,6 @@ TEST_F(ResourceBindingAnalysisTest, TestUnboundedAndOverlap) {
   // StructuredBuffer<float> C[]  : register(t0, space2);
   // StructuredBuffer<float> D    : register(t4, space2); /* overlapping */
   StringRef Assembly = R"(
-%__cblayout_CB = type <{ i32 }>
```

Comment:
Removing `%__cblayout_CB` because it is not used (copy & paste error from previous change). Same thing on line 201.